test/services/task_list_merge_test.dart

```diff
@@ -0,0 +1,189 @@
+import 'dart:convert';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:p2p_task/models/task.dart';
+import 'package:p2p_task/models/task_list.dart';
+import '../utils/device_task_list.dart';
+
+void main() {
+  var devices = <DeviceTaskList>[];
+
+  setUp(() async {
+    devices = [
+      await DeviceTaskList.create(name: 'device1'),
+      await DeviceTaskList.create(name: 'device2'),
+    ];
+  });
+
+  test('distinct task lists', () async {
```

Comment:
is this `test('multiple task lists keep their tasks distinct', ...` or test multiple task lists don't mix their tasks

Reply:
I agree that this test is not very good. It is there to test if the task lists that are created in setup() are really distinct, but it is more a test of the memory database than a test of the task list merging.

I originally added this test because I got unexpected errors at the beginning caused the databases from the different test devices not being distinct.

Should I just delete this whole test or move it into a separate file?